apps/sumologic/src/app/install/page.tsx

```diff
@@ -0,0 +1,214 @@
+    <div className={styles.container}>
+      <div className={styles.modal}>
+        <div className={styles.timeline_container}>
+          <Step
+            active={active === '1'}
+            number="1"
+            onClick={() => {
+              setActive('1');
+            }}
+            text="Generate API Key"
+          />
+          <div className={styles.timeline} />
+          <div className={styles.timeline} />
+          <Step
+            active={active === '2'}
+            number="2"
+            onClick={() => {
+              setActive('2');
+            }}
+            text="Link Application"
+          />
+        </div>
+        {active === '1' && (
+          <InstructionItems
+            active="1"
+            heading="Create API Key"
+            instructions={[
+              ', click your name in the left-nav and open the Preferences page.',
+              'In the My Access Keys section, click + Add Access Key.',
+              'The Create a Sumo Logic Access Key window appears.',
+              'Enter a name for the access key in the Name field.',
+              'Copy the Access ID and Access Key',
+            ]}
+          />
+        )}
+        {active === '2' && (
+          <>
+            <InstructionItems
+              active="2"
+              heading="Link Application"
+              instructions={['Paste your Access ID and Access Key from your application below:']}
+            />
+            <form action={formAction} className={styles.formContainer}>
+              <div className={styles.inputFields}>
+                <div role="group">
+                  <label htmlFor="token">Access ID</label>
+                  <input
+                    id="accessId"
+                    minLength={1}
+                    name="accessId"
+                    placeholder="Paste Your Access ID"
+                    type="text"
+                  />
+                  {state.errors?.accessId?.at(0) ? (
+                    <span>{state.errors.accessId.at(0)}</span>
+                  ) : null}
+                </div>
+                <div role="group">
+                  <label htmlFor="accessKey">Access Key</label>
+                  <input
+                    id="accessKey"
+                    minLength={1}
+                    name="accessKey"
+                    placeholder="Paste Your Access Key"
+                    type="text"
+                  />
+                  {state.errors?.accessKey?.at(0) ? (
+                    <span>{state.errors.accessKey.at(0)}</span>
+                  ) : null}
+                </div>
+                <div role="group">
+                  <label htmlFor="sourceRegion">Deployment Region</label>
+                  <select
+                    id="sourceRegion"
+                    name="sourceRegion"
+                    onChange={(e) => {
+                      setSelectedDomain(e.target.value);
+                    }}
+                    value={selectedDomain}>
```

Comment:
```suggestion
                    >
```
I don't think we need to store the select value in a state.

Reply:
Okay.